### ISSUE TITLE
TODOs: List of simple background theory pages (needed for sphere)

### ISSUE THREAD

**Issue by dccowan** [geoscixyz/em#98]
- Theory of Frequency to Time domain transforms (inverse Fourier, inverse Laplace, inverse sine/cosine)
- Self-demagnetization page
- Frequency-dependent wave attenuation
- The field from a magnetic dipole

**Comment by lheagy** [comment]
- Time to freq : Maxwell 1 / Physical Principles / Principles of EM induction / Going between Time and Frequency (see #99 ) 
- self de-mag : maybe with Maxwell 2 / Mag (not there yet!) / sphere (also not there yet, ha)
- freq dependent wave attenuation: Maxwell 1 / Physical Principles / Principles of EM induction / Frequency domain (see #99) 
- field from a magnetic dipole: Maxwell 1 Fundamentals / Physical Principles / Sources in a homogeneous medium / magnetic dipole (see #99) 

What do you think of these?

**Comment by lheagy** [comment]
@dccowan: could you take a look at this issue and see if you think it still needs to be addressed?

**Comment by dccowan** [comment]
- The time to frequency domain transform (Fourier, Laplace, sine/cosine) could go in the appendix section we made for fundamentals.
- I realized that self-demagnetization is a potential fields issue and not really something that comes up in EM problems (unless someone can provide me an example). I think we can ignore this for the time being.
- The full expression for frequency-dependent attenuation and skin depth is provided for plane waves in homogeneous media.
- Dipole content is super comprehensive.

The only remaining item is the time to frequency domain content. I will create a separate issue for this.
